## Hypthesis testing assignment Q1

A F&B manager wants to determine whether there is any significant difference in the diameter of the cutlet between two units. A randomly selected sample of cutlets was collected from both units and measured? Analyze the data and draw inferences at 5% significance level. Please state the assumptions and tests that you carried out to check validity of the assumptions.

Minitab File : Cutlets.mtw


In [1]:
#Import the required libraris required
import pandas as pd
import scipy.stats as stats

In [2]:
#Import the data
data = pd.read_csv('Cutlets.csv')
data.head()

Unit A  Unit B
0  6.8090  6.7703
1  6.4376  7.5093
2  6.9157  6.7300
3  7.3012  6.7878
4  7.4488  7.1522

H0 (Null Hypothesis): There is no change in diameter of cutlet between two units

H1(Alternative Hypothesis): There is a change in diameter of cutlet between two units

Given 2 indipendent samples and population statndard devaition is not known hence using the 2 sample t test(Individual)

In [3]:
Unit_A = data.iloc[:,0]
Unit_B = data.iloc[:,1]

In [4]:
#Using a 2-tailed 2 sample t-test
Tstat, pval= stats.ttest_ind(Unit_A, Unit_B)
Tstat, pval

(0.7228688704678061, 0.4722394724599501)

Conclusion: 
p value(0.4722) > (α = 0.05)
Hence failed to reject the null hypothesis. 
So there is no significant difference in the diameter of cutlet between the two units.